src/libxtp/gwbse/bse.cc

```diff
@@ -212,25 +201,47 @@ tools::EigenSystem BSE::solve_hermitian(BSE_OPERATOR& h) const {
   return result;
 }
 
+
 tools::EigenSystem BSE::Solve_singlets_BTDA() const {
-  SingletOperator_BTDA_ApB Hs_ApB(_epsilon_0_inv, _Mmn, _Hqp);
-  configureBSEOperator(Hs_ApB);
```

Comment:
After the benchmarks and such I would want to remove the whole LAPACK stuff for the BSE.

Reply:
I would keep it as a back up ...